examples/autoquality/autoquality_usage.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<p>I would add some titles, like</p><p> </p><pre class="ql-syntax" spellcheck="false"># Project setup

Let's create an appropriate Toloka project, a training pool and a base pool(a pool with a basic settings that will be cloned by autoquality)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Toloka/toloka-kit/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>